services/spar/test-integration/Test/Spar/Scim/UserSpec.hs

```diff
@@ -1780,13 +1780,14 @@ testUpdateUserRole = do
   let galley = env ^. teGalley
   (owner, tid) <- call $ createUserWithTeam brig galley
   tok <- registerScimToken tid Nothing
-  let mTargetRoles = Nothing : map Just [minBound ..]
+  let mRoles = Nothing : map Just [minBound @Role ..]
```

Comment:
I think this type annotation is not needed, is it a left over or is it meant for documenting the code?

Reply:
both! :)